packages/x-components/static-docs/experience-search-and-discovery/my-history.md

```diff
@@ -0,0 +1,50 @@
+---
+title: My History UI
+---
+
+The My History UI component displays the complete list of history queries, allowing shoppers to reuse the search queries they performed once and find the products they liked in the past but now are unable to find.  
+
+![My History](~@assets/media/overview_myhistory.svg)
+
+::: interact
+
+Can't quite capture the concept? Learn more about [My History](/explore-empathy-platform/features/my-history-overview.md). 
+
+:::
+
+From the frontend perspective, My History is a UI component that specifically manages the display of a list of history search suggestions. To display a dashboard featuring shoppers' data privacy and control, My History is combined with other UI elements and components, such as a toggle switch for shoppers to enable or disable search history tracking.
```

Comment:
Tracking again...this is not a blocker, just think if you find a better word.

Reply:
We can just remove tracking and guess it also works: `... to enable or disable search history.`

Reply:
Works for me